apis/placement/v1beta1/resourcesnapshot_types.go

```diff
@@ -22,8 +22,11 @@ const (
 	// NumberOfResourceSnapshotsAnnotation is the annotation that contains the total number of resource snapshots.
 	NumberOfResourceSnapshotsAnnotation = fleetPrefix + "numberOfResourceSnapshots"
 
-	// ResourceSnapshotNameFmt is resourcePolicySnapshot name format: {CRPName}-{resourceIndex}.
-	ResourceSnapshotNameFmt = "%s-%d"
+	// SubindexOfResourceSnapshotAnnotation is the annotation to store the subindex of resource snapshot in the group.
+	SubindexOfResourceSnapshotAnnotation = fleetPrefix + "subindexOfResourceSnapshot"
+
+	// ResourceSnapshotNameFmt is resourcePolicySnapshot name format: {CRPName}-{resourceIndex}-snapshot.
```

Comment:
Maybe add a comment here to explain that this is for temporary usage?

Reply:
it's not temporary. Added another format to address your concern.